### PR TITLE
CNV-18515: CNV 4.13 release note shell

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 
4.13

Issue:
[CNV-18515](https://issues.redhat.com//browse/CNV-18515)

Link to docs preview:
https://55234--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html

- [ ] QE has approved this change. N/A

### CODE REVIEW

**Comment by aspauldi** [comment]
@sjhala-ccs

**Comment by aspauldi** [comment]
@sjhala-ccs, can you please review the updates? I removed New and changed features (except one marked as recurring) and added placeholders for the RNs (which you can't see because they are commented out). Thanks!

File: virt/virt-4-13-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+
+[id="virt-4-13-web-new"]
+=== Web console
+
+
+
+
+//NOTE: Comment out deprecated and removed features (and their IDs) if not used in a release
+//[id="virt-4-13-deprecated-removed"]
```
Comment (sjhala-ccs) [inline virt/virt-4-13-release-notes.adoc:98]: Uncomment lines 98 and 99

File: virt/virt-4-13-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+//CNV-16390 Supported guest operating systems
+To view the supported guest operating systems for {VirtProductName}, refer to link:https://access.redhat.com/articles/973163#ocpvirt[Certified Guest Operating Systems in Red Hat OpenStack Platform, Red Hat Virtualization and OpenShift Virtualization].
+
+
+[id="virt-4-13-new"]
+== New and changed features
+
+//CNV-18488 SVVP for 4.12: Ensure platform passes Windows Server Virtualization Validation Program - with RHCOS workers
+//NOTE: This is a recurring release note. Modify the existing note text below if recommended by QE.
+* OpenShift Virtualization is certified in Microsoft's Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
```
Comment (sjhala-ccs) [inline virt/virt-4-13-release-notes.adoc:43]: s/OpenShift Virtualization/{VirtProductName}

File: virt/virt-4-13-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+* You can now use Microsoft Windows 11 as a guest operating system. However, {VirtProductName} {VirtVersion} does not support USB disks, which are required for a critical function of BitLocker recovery. To protect recovery keys, use other methods described in the link:https://learn.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-recovery-guide-plan[BitLocker recovery guide].
+
+//CNV-19145 Support live migration policies
+* You can create live migration policies with specific parameters, such as bandwidth usage, maximum number of parallel migrations, and timeout, and apply the policies to groups of virtual machines by using virtual machine and namespace labels.
+
+
+[id="virt-4-13-bug-fixes"]
+== Bug fixes
+
+*
```
Comment (sjhala-ccs) [inline virt/virt-4-13-release-notes.adoc:205]: Delete the * on lines 205, 206, and 207

File: virt/virt-4-13-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS 8__.
+** Intel and AMD CPUs.
+
+
+//CNV-18462 Release note: CHANGE  Retire alpha APIs for Storage
+
+//CNV-19043 Release note: CHANGE Remove rhel6 support
```
Comment (sjhala-ccs) [inline virt/virt-4-13-release-notes.adoc:53]: Should CNV-19043 and CNV-18462 be moved to the **Removed features** section?

File: virt/virt-4-13-release-notes.adoc [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+
+//CNV-21838 Release note: NEW OpenShift Virtualization supports jumbo frames with OVN Kubernetes
```
Comment (sjhala-ccs) [inline virt/virt-4-13-release-notes.adoc:55]: A few of these RNs could probably go under their component headings to spread them out a bit. For example, CNV-21838 is a Networking RN, CNV-23474 is a Web console RN.

**Review by sjhala-ccs — COMMENTED** [review]
A few minor things, otherwise LGTM. Thanks, Audrey!

File: virt/virt-4-13-release-notes.adoc (hunk 1 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:98]: done!

File: virt/virt-4-13-release-notes.adoc (hunk 2 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:43]: What does this mean? s/OpenShift Virtualization/{VirtProductName}

File: virt/virt-4-13-release-notes.adoc (hunk 3 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:205]: done!

File: virt/virt-4-13-release-notes.adoc (hunk 4 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:53]: done!

File: virt/virt-4-13-release-notes.adoc (hunk 5 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:55]: I moved those and a couple of others around; open to suggestions if you feel other ones should move, too,

File: virt/virt-4-13-release-notes.adoc (hunk 2 above)
Comment (aspauldi) [reply virt/virt-4-13-release-notes.adoc:43]: I looked at another file and copied what I saw in place of OpenShift Virtualization there {VirtProductName} to this file. Is that correct?

Comment (sjhala-ccs) [reply virt/virt-4-13-release-notes.adoc:43]: It means replace "OpenShift Virtualization" with the attribute `{VirtProductName}`. The `{VirtProductName} `attribute is defined in the `common-attributes.adoc` file and resolves to the name of our product when the docs are rendered.

**Comment by aspauldi** [comment]
I've incorporated your commments; thanks again.

**Comment by aspauldi** [comment]
@sjhala-ccs

**Pull request merged**